src/prasad/Whileloop.java

```diff
@@ -0,0 +1,54 @@
+package prasad;
+
+public class Whileloop {
+	void evennumber() {
+		int index=10;
+		while(index<=15) {
+			if(index%2==0)
+				System.out.println(index);
+			index++;
+		}
+	}
+	
+	void divisibleby5(int number1,int number2) {
+		int start = number1;
```

Comment:
why do we need this local variables?
this can handle with number1 & number2 as well